assets/javascripts/modules/react/weeklyRenew.jsx

```diff
@@ -108,12 +110,15 @@ class WeeklyRenew extends React.Component {
             } else {
                 //It's a good promotion, but it's not a weekly one
                 this.setState({
+                    promo:'The promotion you have entered is not currently valid for any of the availble payment methods.',
```

Comment:
Thinking about this, maybe rename 'payment methods' to 'billing periods' or 'payment schedules' to not be confusing to the method of payment (DD vs Credit Card)